qtism/data/content/xhtml/html5/Rt.php

```diff
@@ -22,9 +22,9 @@
 
 namespace qtism\data\content\xhtml\html5;
 
-use qtism\data\content\FlowStatic;
+use qtism\data\content\Inline;
 
-class Rt extends Html5LayoutElement implements FlowStatic
+class Rt extends Html5LayoutElement implements Inline
```

Comment:
While `<figure>` and `<ruby>` must indeed be of an inline type, the `<rt>` is still a block element.
```suggestion
class Rt extends Html5LayoutElement
```